incidentresponse/templates/incidents.html

```diff
@@ -3,50 +3,59 @@
 {% load unslackify %}
 
 {% block head %}
-<link rel="stylesheet" href="{% static "incidents.css" %}">
+    <style>
```

Comment:
Long term, will we need to put all CSS within the HTML or do we think they'd be a way to keep these separate? I believe we're seeing this because Django won't serve static whiles when not in development/debug mode as it sees that as a security risk and inefficient. I believe the path is to use Ngnix or something similar.

I came across a comment on the Monzo repo this week which mentioned they're no longer using the built in UI, and instead have their own UI that consumers the Response API instead. I wonder if long term, we should iterate towards a similar model (when requirements force us too -- for example, displaying and editing action points in the UI and would be a good reason to build a UI in Blazor 😄).

Reply:
I believe the problem was within the fact we're extending another Django app, and we couldn't resolve our static directory over theirs. In-lining was the simplest solution to the problem.